DigitalLearningSolutions.Data/DataServices/BrandsDataService.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.Common;
+
+    public interface IBrandsDataService
+    {
+        IEnumerable<BrandDetail> GetAllBrands();
+    }
+
+    public class BrandsDataService : IBrandsDataService
+    {
+        private const string BrandsSql =

```

Comment:
I would just inline this - it's unlikely to be used in another method, and having it inlined means it's quicker to see when you're reading through the method.

Reply:
We actually do use it again in the next ticket where we only get 1 brand.

Reply:
Ah fair enough. Can you make the name a bit more descriptive so it's more obvious what you're doing with the brands? Something like "SelectAllBrandsSql"?